src/components/dashboard/FaceVerification/components/DeviceOrientationError.js

```diff
@@ -33,7 +33,7 @@ const DeviceOrientationError = ({ styles, displayTitle, onRetry, exception }) =>
     <Wrapper>
       <View style={styles.topContainer}>
         <Section style={styles.descriptionContainer} justifyContent="space-evenly">
-          <Section.Title fontWeight="normal" textTransform="none" color="red">
+          <Section.Title fontWeight="regular" textTransform="none" color="red">
```

Comment:
doesn't it breaks current look at web/android ? 
maybe the issue is related to the Roboto font not loading on iOS ? **please re check this**. 

if you'd need some platform-specific, define & set style for the element, then use Platform.select in getStylesFromProps

Reply:
FONT_WEIGHTS doesn't include 'normal', it is not a platform-specific issue

Reply:
how it worked before ? how it works on web and Android ?

Reply:
On web it works the same